.github/workflows/python_test.yml

```
@@ -0,0 +1,35 @@
+name: Python package
```

Keep a name that matches the file name
```suggestion
name: Python test
```